assets/javascripts/kitten/components/buttons/button.js

```diff
@@ -22,8 +22,13 @@ export class Button extends React.Component {
 
     const Tag = tag
 
+    // Adds keyboard accessibility to `<a>`
+    const tabindex = this.props.tag == "a" && !this.props.href ? 0 : null
```

Comment:
Le && a plus de précédence que le ?. Donc en fait il faut le lire comme `(this.props.tag == "a" && !this.props.href) ? 0 : null`.